versioned_docs/version-1.2.x/surrealql/statements/create.mdx

```diff
@@ -25,58 +25,143 @@ CREATE [ ONLY ] @targets
+CREATE person:100
+```
 
--- Create a new record with a specific string id
-CREATE person:tobie SET name = 'Tobie', company = 'SurrealDB', skills = ['Rust', 'Go', 'JavaScript'];
+The above will create a new record with the ID `person:100`.
 
--- Create just a single record
--- Using the ONLY keyword, just an object for the record in question will be returned.
--- This, instead of an array with a single object.
-CREATE ONLY person:tobie SET name = 'Tobie', company = 'SurrealDB', skills = ['Rust', 'Go', 'JavaScript'];
+```bash title="Response"
+[
+	{
+		"id": "person:100"
+	}
+]
 ```
 
-Instead of specifying record data using the `SET` clause, it is also possible to use the `CONTENT` keyword to specify the record data using a SurrealQL object.
+It is also possible to specify the ID of the record you want to create using a string or any of the supported formats for [record IDs](/docs/surrealdb/surrealql/datamodel/ids).
+
+
+
+## Adding Record Data
+
+When creating a record, you can specify the record data using the `SET` clause, or the `CONTENT` clause. The `SET` clause is used to specify the record data using a list of key-value pairs, while the `CONTENT` clause is used to specify the record data using a SurrealQL object. The `CONTENT` clause is useful when the record data is already in the form of a SurrealQL or JSON object.
 
 ```surql
 -- Create a new record with a random id
-CREATE person CONTENT {
-	name: 'Tobie',
-	company: 'SurrealDB',
-	skills: ['Rust', 'Go', 'JavaScript'],
-};
+CREATE person:100 SET name = 'Tobie', company = 'SurrealDB', skills = ['Rust', 'Go', 'JavaScript'];
+```
+
+The above will create a new record with the ID `person:100` and the specified data.
+
+```bash title="Response"
+[
+	{
+		"id": "person:100",
+		"name": "Tobie",
+		"company": "SurrealDB",
+		"skills": ["Rust", "Go", "JavaScript"]
+	}
+]
+```
 
--- Create a new record with a specific id
-CREATE person:tobie CONTENT {
+Alternatively, you can specify the record data using the `CONTENT` keyword.
+
+```surql
+-- Create a new record with a numeric id
+CREATE person:100 CONTENT {
 	name: 'Tobie',
 	company: 'SurrealDB',
 	skills: ['Rust', 'Go', 'JavaScript'],
 };
 ```
+
+## Clasues and Options
+
+Using the `ONLY` keyword, just an object for the record in question will be returned. This, instead of an array with a single object.
+
+```surql
+-- Create just a single record
+CREATE ONLY person:tobie SET name = 'Tobie', company = 'SurrealDB', skills = ['Rust', 'Go', 'JavaScript'];
+```
+### Return Values
+
 By default, the create statement returns the record value once the changes have been made. To change the return value of each record, specify a `RETURN` clause, specifying either `NONE`, `BEFORE`, `AFTER`, `DIFF`, or a comma-separated list of specific fields to return.
 
 ```surql
--- Don't return any result
-CREATE person SET age = 46, username = "john-smith" RETURN NONE;
+CREATE person:20 SET age = 46, username = "john-smith" RETURN NONE;
```

Comment:
I've removed the number since it doesn't seem like it will make a difference if it's randomly generated 🤔 ?

Also just keeping in mind that using numbers as ids is not something we want to encourage since we don't have a good way of doing these typical auto increment of numbers like in relational dbs

Reply:
My thought here was  the reader will copy the example but I agree with this in mind no ID would make more sense plus we don't want number identifies.